main/webapp/modules/core/scripts/index/parser-interfaces/xml-parser-ui.js

```diff
@@ -31,6 +31,8 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
  */
 
+var recordElementSelected = false;
```

Comment:
I think it'd be cleaner if this was not a global variable, I would store it as an attribute of the importer.